public/apps/configuration/panels/role-list.tsx

```diff
@@ -137,6 +139,22 @@ export function RoleList(props: AppDependencies) {
     fetchData();
   }, []);
 
+  const handleDelete = async () => {
+    const rolesToDelete: string[] = selection.map(r => r.role_name);
+    try {
+      await requestDeleteRoles(props.coreStart.http, rolesToDelete);
+      // Refresh from server (calling fetchData) does not work here, the server still return the roles
```

Comment:
better to check backend when setup template setting for the role index whether it set the refresh interval to minimal value , i remember it will be 1s.
https://www.elastic.co/guide/en/elasticsearch/reference/master/index-modules.html#index-refresh-interval-setting

Reply:
If server returns 200 success, it should be correct that just delete from local memory. No need to let user wait for 1 more second. And waiting for constant 1 second could still be inconsistent if user changed the settings.